### ISSUE TITLE
Validation happens at parse time, resulting in no validation for cards constructed via the object model.

### ISSUE THREAD

**Issue by RebeccaAnne** [Microsoft/AdaptiveCards#2811]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* Android
* iOS
* UWP

# Issue

All card validation happens at parse time, which means that cards constructed by the object model directly in code (not via parsing json) may be invalid in a number of ways. Consider whether we want to move towards object model validation.

**Comment by dclaux** [comment]
That is true of the TS renderer as well, and is by design. Cards constructed via the object model should not be subject to validation IMO.
